src/main/java/org/veupathdb/service/eda/data/plugin/selfcorrelation/SelfCorrelationUnipartitenetworkPlugin.java

```diff
@@ -66,10 +66,16 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
     ByteArrayInputStream statsIn = new ByteArrayInputStream(statsBytes.toByteArray());
     dataStreams.put("statsFile.json", statsIn);
 
+    // some default filtering thresholds
+    Number correlationCoefThreshold = getPluginSpec().getCorrelationCoefThreshold() != null ? getPluginSpec().getCorrelationCoefThreshold() : 0.2;
```

Comment:
I'd rather not set a default threshold (or just set a default threshold of 0) if we're not returning the threshold value to the frontend. Otherwise whomever asking for the data won't realize they're missing a bunch!

Reply:
interesting... you may have found a reason to move it to plot.data